packages/vm/src/errors/mod.rs

```diff
@@ -9,6 +9,8 @@ pub use region_validation_error::RegionValidationError;
 pub use vm_error::VmError;
 
 pub type CommunicationResult<T> = core::result::Result<T, CommunicationError>;
-pub type FfiResult<T> = core::result::Result<T, FfiError>;
+/// A return element and the gas cost of this FFI call
+pub type FfiSuccess<T> = (T, u64);
```

Comment:
Given the discussion below, this interface can easily be misleading. Sometimes the second component is the gas charged by the SDK and sometimes the gas to be charged as part of the VM budget. Will think about how to best represent that.

Reply:
Done in https://github.com/CosmWasm/cosmwasm/pull/475/commits/2c1da17f65e09f21e610d3659bedad76271b10b3